StRoot/StPicoMixedEventMaker/StPicoMixedEventMaker.cxx

```diff
@@ -10,7 +10,6 @@
 #include "StPicoDstMaker/StPicoEvent.h"
 #include "StPicoDstMaker/StPicoTrack.h"
 #include "StPicoDstMaker/StPicoBTofPidTraits.h"
-#include "StPicoPrescales/StPicoPrescales.h"
```

Comment:
Hmm, this looks like an artifact of the rnc-lbl repo (I forked the code from that one)